feats/redis.py

```diff
@@ -0,0 +1,152 @@
+from typing import List
+from redis import Redis
+from redis.exceptions import ConnectionError
+
+
+class NotConnectedError(ConnectionError):
```

Comment:
Raising this sort of error (One rooted in redis.exception) makes it difficult to build a generic interface out of -- the feats app would have no way to catch this exception without importing redis.

What if we moved this error into storage.py & had a less-library specific parent class?